packages/react-slider/src/utils/on.ts

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const on = (element: Element, eventName: string, callback: (ev: any) => void) => {
```

Comment:
Do we not have a standard pattern for handling this in converged? If we don't already, this should potentially go in `react-utilities` (and ideally have a generic for the event type).